### PR TITLE
CB-74: Remove all warnings from test

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
CB-74: [Fix warning on tests](https://issues.openmrs.org/browse/CB-74)

## SUMMARY:
This PR fixes all the warnings that show when running tests

### CODE REVIEW

File: app/js/components/tabs/tabcomponents/patientComponent.jsx [hunk 1]
```diff
@@ -71,24 +72,24 @@ class PatientComponent extends Component {
       }
     }
     // add appropriate birthdate constraints to the search parameters
-    if (startDate && endDate) {
+    if (startDate && endDate) {  
+      // coerce the values of start date and end date to numbers for comparison    
+      let startYear = Number(startDate.split('-')[0]);
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/patientComponent.jsx]: I thought this part was already merged in this pull request https://github.com/openmrs/openmrs-owa-cohortbuilder/pull/93

Comment (jomadoye) [reply app/js/components/tabs/tabcomponents/patientComponent.jsx]: @dkayiwa I have fixed this, sorry for the oversight.

File: app/js/components/tabs/tabsComponent.jsx [hunk 2]
```diff
@@ -55,8 +55,9 @@ class TabsComponent extends Component {
             apiHelper.get(url).then(response => {
                 response.json().then(data => {
                     resolve(data);
-                });
-            });
+                //TODO: This errors need to be handled
```
Comment (dkayiwa) [inline app/js/components/tabs/tabsComponent.jsx:58]: Do you plan to handle these errors?

Comment (jomadoye) [reply app/js/components/tabs/tabsComponent.jsx:58]: Yes, @dkayiwa but not in this ticket. There is currently an open issue that request for the complete refactor of this component [Link to issue](https://issues.openmrs.org/browse/CB-52?jql=project%20%3D%20CB%20and%20status%20!%3D%20closed). Handling this errors should fall under the issue above. Also I would like to recommend we create an issue to handle all components errors in the code base, as currently there is no structure in place to do that.

Comment (dkayiwa) [reply app/js/components/tabs/tabsComponent.jsx:58]: You are free to create more tickets and add to the sprint as you see best. 👍

**Pull request merged**